examples/sample_token_generate_refresh.py

```diff
@@ -0,0 +1,58 @@
+import sys
+
+from uid2_client import Uid2PublisherClient
+from uid2_client import TokenGenerateResponse
+from uid2_client import TokenGenerateInput
+
+
+def _usage():
+    print('Usage: python3 sample_token_generate_refresh.py <base_url> <auth_key> <secret_key>', file=sys.stderr)
+    sys.exit(1)
+
+
+if len(sys.argv) <= 3:
+    _usage()
+
+base_url = sys.argv[1]
+auth_key = sys.argv[2]
+secret_key = sys.argv[3]
+
+
+publisher_client = Uid2PublisherClient(base_url, auth_key, secret_key)
+print("Generating Token")
+token_generate_response = publisher_client.generate_token(TokenGenerateInput.from_email("testpythonsdksampletokengenerate@email.com").do_not_generate_tokens_for_opted_out())
+
+status = token_generate_response.status
```

Comment:
could you add a check if status is not success just print something to tell user whats wrong and just quite instead of trying to get_identity which won't be successful?

Reply:
If the status is not "success" or "optout", the constructor of TokenGenerateResponse will throw an exception, same as is done in java. urllib, however, is also going to throw an exception if an error status code is returned which must be handled. I am putting a try catch into sample_token_generate_refresh to show error handling. Output of the example looks like this now for an error:
```
Generating Token
HTTP Error 401: Unauthorized

Process finished with exit code 1
```